src/containers/following-page/store/sagas.ts

```diff
@@ -3,25 +3,39 @@ import { USER_LIST_TAG } from '../FollowingPage'
 import { put, select } from 'redux-saga/effects'
 import { getId, getUserList, getUserIds } from './selectors'
 import { ID } from 'models/common/Identifiers'
-import AudiusBackend from 'services/AudiusBackend'
 import { createUserListProvider } from 'containers/user-list/utils'
 import { getFollowingError } from './actions'
 import { watchFollowingError } from './errorSagas'
 import User from 'models/User'
 import { getUser } from 'store/cache/users/selectors'
+import apiClient from 'services/audius-api-client/AudiusAPIClient'
+import { encodeHashId } from 'utils/route/hashIds'
 
 const provider = createUserListProvider<User>({
   getExistingEntity: getUser,
   extractUserIDSubsetFromEntity: () => [],
   fetchAllUsersForEntity: ({
     limit,
     offset,
-    entityId
+    entityId,
+    currentUserId
   }: {
     limit: number
     offset: number
     entityId: ID
-  }) => AudiusBackend.getFollowees(entityId, limit, offset),
+    currentUserId: ID | null
+  }) => {
+    const encodedUserId = currentUserId
```

Comment:
maybe we should have a wrapper function around encodeHashID or encodeHashId should take a nullable value so we can avoid this song and dance everywhere